### PR TITLE
Fix/durcom 274/update on change

### PR DESCRIPTION
**Pull request by dpellenwood**
Updates the plugin version and header for Modern Tribe and fixes an issue with image selects not changing on the first change/update in the Gutenberg block editor.

### CODE REVIEW

File: image-select-v5.php [hunk 1]
```diff
@@ -30,11 +30,12 @@ function __construct()
 		$this->settings = array(
 			'path'				=> apply_filters('acf/helpers/get_path', __FILE__),
 			'dir'				=> apply_filters('acf/helpers/get_dir', __FILE__),
-			'version'=> '1.0.0'
+			'version'=> '1.0.2'
 		);
 
 		// do not delete!
 		parent::__construct();
+		parent::__construct();
```
Comment (ryanurban) [inline image-select-v5.php]: This intentional?

File: js/image-select.js [hunk 2]
```diff
@@ -20,10 +18,19 @@
...
+				.find("input[type='radio']")
+				.attr("checked",false)
+				.removeAttr("data-checked");
+
+			jQuery('label[for="' + id + '"]')
+				.addClass('acf-image-select-selected')
+				.find("input[type='radio']")
+				.attr("checked",true)
+				.attr("data-checked","checked")
+				.trigger('change');
```
Comment (ryanurban) [inline js/image-select.js:33]: 🔥

**Review by ryanurban — APPROVED** [review]
Thank you Dave!

File: image-select-v5.php (hunk 1 above)
Comment (dpellenwood) [reply image-select-v5.php]: Nope. Removed.

Comment (ryanurban) [reply image-select-v5.php]: 😂 You got it already 🎖️

**Comment by dpellenwood** [comment]
> Thank you Dave!

All props to @sleddd . She debugged it and submitted the fix.

**Comment by ryanurban** [comment]
Thank you @sleddd! 🥇

**Pull request merged**